multirange.js

```diff
@@ -59,8 +59,38 @@ self.multirange = function(input) {
 		ghost.style.setProperty("--high", input.valueHigh * 100 / max - 1 + "%");
 	}
 
-	input.addEventListener("input", update);
-	ghost.addEventListener("input", update);
+    function passClick(evt) {
+      // Are the ghost and input elements inverted? (ghost is lower range)
+      var isInverted = input.valueLow == ghost.value;
+      // Find the horizontal position that was clicked (as a percentage of the element's width)
+      var clickPoint = evt.offsetX / this.offsetWidth;
+      // Map the percentage to a value in the range (note, assumes a min value of 0)
+      var clickValue = max * clickPoint;
+
+      // Get the distance to both high and low values in the range
+      var highDiff = Math.abs(input.valueHigh - clickValue);
+      var lowDiff = Math.abs(input.valueLow - clickValue);
+
+      if (lowDiff < highDiff && !isInverted || (isInverted && lowDiff > highDiff)) {
+        // The low value is closer to the click point than the high value
+        // We should update the low value input
+        var passEvent = new MouseEvent("mousedown", {screenX: evt.screenX, clientX: evt.clientX});
+        // Pass a new event to the low "input" element (which is obscured by the
+        // higher "ghost" element, and doesn't get mouse events outside the drag handle
+        input.dispatchEvent(passEvent);
+        // The higher "ghost" element should not respond to this event
+        evt.preventDefault();
+        return false;
+      }
+      else {
+      	// The high value is closer to the click point than the low value
+        // The default behavior is appropriate, so do nuthin
+      }
+    }
+
+    ghost.addEventListener("mousedown", passClick);
```

Comment:
If `passClick` is not used anywhere else, I'd rather have it as an anonymous function here, I think it's more readable that way (reader does not have to go following the reference).